projects/kit/components/input-files/input-files.template.html

```diff
@@ -8,7 +8,11 @@
     [active]="pseudoActive"
     [disabled]="computedDisabled"
 >
-    <label automation-id="tui-input-file__label">
+    <label
+        automation-id="tui-input-file__label"
+        (tuiDroppableDragOverChange)="onDragOver($event)"
+        (tuiDroppableDropped)="onDropped($event, {formatRejection, maxSizeRejection})"
```

Comment:
If you moved those directives here you should remove them from the built-in input.

Reply:
fixed